runtime/src/configs/mod.rs

```diff
@@ -502,6 +516,7 @@ impl pallet_storage_providers::Config for Runtime {
     type MaxCommitmentSize = ConstU32<1000>;
     type ZeroSizeBucketFixedRate =
         runtime_params::dynamic_params::runtime_config::ZeroSizeBucketFixedRate;
+    type TopUpGracePeriod = ConstU32<600>; // 1 hour with 6 second timeslots.
```

Comment:
Wasn't this meant to be in the order of days?

Reply:
thanks for spotting, I had put this initially